### PR TITLE
Fix issue #767, duplicate @JsonProperty

### PR DESCRIPTION
**Pull request by glarfs**
There is a duplicated @JsonProperty on the file, could you integrate it? Please

### CODE REVIEW

File: springfox-swagger-common/src/main/java/springfox/documentation/swagger/web/SecurityConfiguration.java [hunk 1]
```diff
@@ -71,7 +71,7 @@ public String getClientSecret() {
     return clientSecret;
   }
 
-  @JsonProperty("apiKey")
```
Comment (glarfs) [inline springfox-swagger-common/src/main/java/springfox/documentation/swagger/web/SecurityConfiguration.java]: apiKey duplicated

**Comment by dilipkrish** [comment]
@glarfs gaah! thats some copy paste error right there! Thanks for catching that... looks like it has failing tests tho'

**Comment by glarfs** [comment]
The test on my local branch(which is pointing to master) without this commit and it fails on the same point, it's strange. Seems that the code which is failing test is another commit.

**Comment by n0mer** [comment]
@glarfs @dilipkrish  there are definitely 2 failing tests

image

**Comment by glarfs** [comment]
There where an issue with tests, Now the tests are fixed

**Comment by dilipkrish** [comment]
Thanks! :+1:

**Comment by glarfs** [comment]
Hi, this fix could be applied also on 2.2.x branch, When are you releasing a new version with this fix?

**Pull request merged**